### DISCUSSION TITLE
Search from suggested or other articles

### DISCUSSION THREAD

**Discussion by purevnorov** [dailydotdev/daily#635] [Feature Requests]
Maybe its Daily.dev's main feature to be not have search bar but sometimes I want to search articles by titles for example: If search bar was avilable I do not need to change my tag setting in order to keep up with JS update. Even if I was checked JS tag what if I want to read JS updates or articles.

**Comment by idoshamun** [comment]
Hi @purevnorov,

Actually, we do have a search bar. Please let me know if this is what you were looking for.
https://app.daily.dev/search
